src/System.Private.CoreLib/src/Internal/Runtime/Augments/RuntimeThread.cs

```diff
@@ -14,22 +14,135 @@ namespace Internal.Runtime.Augments
 {
     public class RuntimeThread : CriticalFinalizerObject
     {
+        /*=========================================================================
```

Comment:
Hmm, there is more code duplicating with CoreFX Thread that I expected

Reply:
Do you have anything specific you have in mind?